src/backend/web/graphql/queries.js

```diff
@@ -0,0 +1,99 @@
+const { gql } = require('apollo-server-express');
+
+const maxPostsPerPage = 30;
```

Comment:
Should this be configurable via `env`?  It seems like we now have 2 places where we do it (here and in the `/posts` pagination code).